hack/quick-install.sh

```diff
@@ -5,12 +5,29 @@ set -o xtrace
 
 export CERT_MANAGER_VERSION=v1.9.1
 export KNATIVE_EVENTING_VERSION=v1.14.3
+export JAEGER_OPERATOR_VERSION=v1.57.0
 
+# Production dependencies
 kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml
 kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
 kubectl rollout status deployment --namespace=cert-manager --timeout=30s
 kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
 
+# Development dependencies
+# Avoid problems on namespace creation when re-executing this file
+kubectl create namespace observability --dry-run=client -o yaml | kubectl apply -f -
+kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/${JAEGER_OPERATOR_VERSION}/jaeger-operator.yaml -n observability
+kubectl rollout status deployment --namespace=observability --timeout=30s
+sleep 5  # The jaeger operator webhook its not fast enough :(
```

Comment:
Since this is a script that developers will likely run and then wait for it to finish, when you have a pause like this it is a good idea to put something on the console so the developers know why the script has "stopped". 

```suggestion
echo "Wait a few seconds for the Jaeger Operator Webhook to finish initialization."
sleep 5
```

Reply:
Sorry, I changed the approach. I was not convinced of adding it on the script, so I deleted the whole installation of the jaeger operator in favor of a deployment with the all-in-one image.